terra-jupyter-aou/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:2.2.11
+FROM us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:2.2.12
```

Comment:
You might also want to remove line 266 https://github.com/DataBiosphere/terra-docker/blob/f43c5aed7e679bb2fa274c421b5a224c804cc4d6/terra-jupyter-aou/Dockerfile#L266
because it is no-longer true after this PR

Reply:
Good catch! I didn't know we did this.

I removed the line and force-pushed to keep the modification commits before the version bump commit.

I searched for other uses of cromshell in the repo and didn't find anything.